src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -44,6 +46,9 @@ export default function (chrome, internals) {
         onRouteChange();
 
         // and some local values
+        $scope.$watchCollection(() => $http.pendingRequests, (pendingHttpRequests = []) => {
+          remove(pendingHttpRequests, isSystemApiRequest);
+        });
```

Comment:
I know I could've shortened this to a single line like this:

```js
$scope.$watchCollection(() => $http.pendingRequests, (pendingHttpRequests = []) => remove(pendingHttpRequests, isSystemApiRequest));
```

But I chose to break this up over multiple lines purely for readability.